docs/releases/6.1.txt

```diff
@@ -86,6 +86,33 @@ Python-level options, as Django does not need to load objects before deleting
 them. As a consequence, the :attr:`~django.db.models.DB_CASCADE` option does
 not trigger the ``pre_delete`` or ``post_delete`` signals.
 
+
+Multi-line template tags and variables
+--------------------------------------
+
+A new setting :setting:`TEMPLATE_TAGS_MULTILINE` is available to switch to a
+multi-line mode for the template engine. If this is turned on you can use
+newlines inside template tags and variable blocks to improve readability:
+
+.. code-block:: html+django
+
+    {%
+        load
+            static
+            i18n
+    %}
+
+    {{
+        some_variable
+        | filter1
+        | filter2
+    }}
+
```

Comment:
I'd briefly mention that comments still do not allow newlines.

Reply:
I guess just remove that section?

Reply:
Yeah, removing is fine, and annotated the section you edited with a `versionchanged` directive.